### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by kimjammer** [KoljaB/RealtimeSTT#57] [General]
Hi there, thanks for your work. I can see that you upgraded the faster-whisper version to 1.0.2 in the main branch since the last release, which fixes a dependency conflict for me. It's been a little bit since the last versioned release, hope to see another release soon! Thanks.

**Comment by KoljaB** [comment]
Any ideas for a new release? Anybody want to see [openWakeWord](https://github.com/dscripka/openWakeWord) support?

**Reply by kimjammer** [reply]
I didn't mean to pressure you for new features, sorry! Having a new release would just let me use the fixed dependencies. I wouldn't say no to openWakeWord though.

**Reply by tgdonovan** [reply]
I would like a different wake word, but it isn’t a high priority for me
